themes/light/templates/schedule.mako

```diff
@@ -38,70 +38,74 @@ const startVue = () => {
             }
         ),
         mounted() {
-            const { $store, $route, config, layout } = this;
+            const { $store, $route } = this;
             $store.dispatch('getConfig');
             $store.dispatch('getShows');
 
-            if (layout === 'list') {
-                const sortCodes = {
-                    date: 0,
-                    show: 2,
-                    network: 5
-                };
-                const sort = config.comingEpsSort;
-                const sortList = (sort in sortCodes) ? [[sortCodes[sort], 0]] : [[0, 0]];
+            const unwatch = this.$watch('layout', () => {
+                unwatch();
```

Comment:
Shouldn't this be outside of the function itself? 😕

Reply:
I wasn't sure if it works this way. My point is to unwatch as soon as I can because I don't want the code to run more than once.

Reply:
Oh it should, just checked the docs and $this.watch returns the function needed to unregister it.